pmdarima/compat/sklearn.py

```diff
@@ -0,0 +1,53 @@
+# -*- coding: utf-8 -*-
+#
+# Author: Charles Drotar <drotarcharles@gmail.com>
+#
+# Patch backend for sklearn
+
+from __future__ import absolute_import
+
+import sklearn
+from sklearn.utils.validation import check_is_fitted
+
+__all__ = [
+    'get_compatible_check_is_fitted'
+]
+
+
+def get_compatible_check_is_fitted(estimator, attributes=None):
+    """Make the ``check_is_fitted`` function compatible across the
+    sklearn 0.21 to 0.22+ boundary.
+
+
+    Parameters
+    ----------
+    estimator : estimator instance,
+        The estimator that will be checked to see if it is fitted.
+
+    attributes : str, optional (default=None)
+        The attributes to be passed into the ``check_is_fitted`` function.
+
+    """
+
+    # Split the version string from sklearn and use that determine how to use
+    # the `sklearn.utils.validation import check_is_fitted` function
+    semantic_version_split = [int(x) for x in sklearn.__version__.split(".")]
+    major_version = semantic_version_split[0]
+    minor_version = semantic_version_split[1]
```

Comment:
You could probably do this a bit more elegantly like:

```python
>>> import pkg_resources
>>> pkg_resources.parse_version(sklearn.__version__)
<Version('0.21.2')>
```

It's nice because it allows comparisons like

```python
if parse_version(sklearn.__version__) >= parse_version('0.22.0')
```

Reply:
This would also prevent breakage from anyone running a local version of scikit that may be versioned, e.g., `0.24.0.dev0`

Reply:
Ahhh good call! Will do!